### PR TITLE
THREE.js: Replace Vector3 with Vector2 for lathe geometry type

### PR DESCRIPTION
**Pull request by johnmarinelli**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/mrdoob/three.js/blob/master/src/geometries/LatheGeometry.js#L89
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by dhritzkiv — CHANGES_REQUESTED** [review]
For the "Provide a URL to documentation or source code" item, could you instead provide a link the the source code of three.js, where lathe geometry uses Vector2?

File: types/three/three-core.d.ts [hunk 1]
```diff
@@ -6644,7 +6644,7 @@ export class IcosahedronGeometry extends PolyhedronGeometry {
 }
 
 export class LatheBufferGeometry extends BufferGeometry {
-    constructor(points: Vector3[], segments?: number, phiStart?: number, phiLength?: number);
+    constructor(points: Vector2[], segments?: number, phiStart?: number, phiLength?: number);
 
     parameters: {
         points: Vector3[];
```
Comment (dhritzkiv) [inline types/three/three-core.d.ts]: Should `points` here  and at L6661 also be `Vector2[]`?

Comment (johnmarinelli) [reply types/three/three-core.d.ts]: Yep - `LatheGeometry` docs also specify Vector2.  Under the hood, `LatheGeometry` uses `LatheBufferGeometry`.

https://threejs.org/docs/#api/geometries/LatheGeometry
https://threejs.org/docs/#api/geometries/LatheBufferGeometry

Comment (johnmarinelli) [reply types/three/three-core.d.ts]: Updated source doc link

Comment (dhritzkiv) [reply types/three/three-core.d.ts]: Cool, thanks!

Do you mind making the change at L6650 and L6651 from `points: Vector3[];` to `points: Vector2[];`?

Comment (johnmarinelli) [reply types/three/three-core.d.ts]: nice catch :) updated

**Review by dhritzkiv — APPROVED** [review]

**Pull request merged**